yapapi/engine.py

```diff
@@ -501,52 +501,24 @@ async def _enforce_debit_note_intervals(self, job: "Job", debit_note: DebitNote)
 
     async def _process_debit_notes(self) -> None:
         """Process incoming debit notes."""
+        max_concurrent_notes = 10
```

Comment:
why is this value hardcoded in the middle of the code? if we need such arbitrary values, let's have them defined as `Final` and defined in a constant-like manner in at the top of the file, as is customary with such constants